ci-operator/jobs/openshift/console/openshift-console-master-presubmits.yaml

```diff
@@ -362,3 +362,26 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
```

Comment:
Doing this will mean that none of our tooling will update the job since it's not generated. If possible, could you add this to your `ci-operator` config instead?

```yaml
base_images:
  shellcheck:
    cluster: https://api.ci.openshift.org
    name: shellcheck
    namespace: ci
    tag: stable
raw_steps:
- source_step:
    clonerefs_image:
      name: clonerefs
      namespace: ci
      tag: latest
    clonerefs_path: /app/prow/cmd/clonerefs/app.binary.runfiles/io_k8s_test_infra/prow/cmd/clonerefs/linux_amd64_pure_stripped/app.binary
    from: shellcheck
    to: shellcheck-src
tests:
- as: verify
  commands: ./shellcheck.sh
  container:
    from: shellcheck-src
```

While also adding the ShellCheck image to an imagestream in `core-services/supplemental-ci-images/` ?